src/components/MasterPlaylist.tsx

```diff
@@ -63,17 +63,17 @@ export default function MasterPlaylist(props: { playlist: PlaylistObj, usedTrack
       .map((t: TrackObj) => t.genres)
       .flat()
       .forEach((g: string) => map.set(g, (map.get(g) ?? 0) + 1));
-      
+
     var mapAsc = Array.from(map.entries()).sort((a, b) => b[1] - a[1]);
     mapAsc.splice(3, 99999);
```

Comment:
dont know whose code this is but the hard coded 99999 screams "break me" lmao. Doubt itll be a problem tho

Reply:
It's mine from a while ago. I wasn't bothered to look up the ts equivalient of `Integer.MAX_VALUE`.

It's been fixed